ui-backend/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/query/cql/SourceWarningsFilterManagerTest.java

```diff
@@ -0,0 +1,191 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.query.cql;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import ddf.catalog.operation.ProcessingDetails;
+import ddf.catalog.security.SourceWarningsFilter;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Test;
+
+public class SourceWarningsFilterManagerTest {
```

Comment:
✏️ If you want more concise tests, annotate the test class with `@RunWith(MockitoJUnitRunner.class)` which will allow you to declare class-level mocks that get reset after every test, like so: 
```
@RunWith(MockitoJUnitRunner.class)
public class SourceWarningsFilterManagerTest {

@Mock private SourceWarningsFilter mockedFilter;
@Mock private ProcessingDetails mockedDetails;

...
```
If you find yourself recreating a lot of the same mocks using the `mock(<class>)` function, this can help.